### ISSUE TITLE
SVG failing (exceptions related to accessibility and batik)

### ISSUE THREAD

**Issue by jwr** [clj-pdf/clj-pdf#199]
clj-pdf 2.5.1 on Ubuntu 18.04.4 LTS. Discovered this after a big migration, so I'm not yet sure what exactly started causing this, still trying to find the cause (I upgraded from 2.4.9 to 2.5.1 recently, as well as moved from Ubuntu 16 to Ubuntu 18).

```
java.awt.AWTError: Assistive Technology not found: org.GNOME.Accessibility.AtkWrapper stack trace:
 java.awt.AWTError: Assistive Technology not found: org.GNOME.Accessibility.AtkWrapper
 at java.awt.Toolkit.loadAssistiveTechnologies (Toolkit.java:807)
    java.awt.Toolkit.getDefaultToolkit (Toolkit.java:886)
    org.apache.batik.bridge.CursorManager.<clinit> (CursorManager.java:102)
    org.apache.batik.bridge.BridgeContext.<init> (BridgeContext.java:1162)
    org.apache.batik.bridge.BridgeContext.<init> (BridgeContext.java:304)
    clj_pdf.section.svg$make_ctx.invokeStatic (svg.clj:10)
    clj_pdf.section.svg$make_ctx.invoke (svg.clj:10)
    clj_pdf.section.svg$_render.invokeStatic (svg.clj:28)
    clj_pdf.section.svg$_render.invoke (svg.clj:24)
    clj_pdf.section.svg$fn__15479.invokeStatic (svg.clj:36)
    clj_pdf.section.svg/fn (svg.clj:32)
    clojure.lang.MultiFn.invoke (MultiFn.java:239)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.AFn.applyTo (AFn.java:144)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$apply.invoke (core.clj:660)
    clj_pdf.section$make_section.invokeStatic (section.clj:43)
    clj_pdf.section$make_section.invoke (section.clj:17)
    clj_pdf.core$append_to_doc.invokeStatic (core.clj:112)
    clj_pdf.core$append_to_doc.invoke (core.clj:107)
    clj_pdf.core$add_item.invokeStatic (core.clj:484)
    clj_pdf.core$add_item.invoke (core.clj:480)
    clj_pdf.core$write_doc.invokeStatic (core.clj:514)
    clj_pdf.core$write_doc.invoke (core.clj:502)
    clj_pdf.core$pdf.invokeStatic (core.clj:571)
    clj_pdf.core$pdf.invoke (core.clj:559)
```

```
java.lang.NoClassDefFoundError: Could not initialize class org.apache.batik.bridge.CursorManager stack trace:                                                                                                                                                                                               java.lang.NoClassDefFoundError: Could not initialize class org.apache.batik.bridge.CursorManager                                                                                                      at org.apache.batik.bridge.BridgeContext.<init> (BridgeContext.java:1162)                                                                                                                                org.apache.batik.bridge.BridgeContext.<init> (BridgeContext.java:304)                                                                                                                                 clj_pdf.section.svg$make_ctx.invokeStatic (svg.clj:10)                                                                                                                                                clj_pdf.section.svg$make_ctx.invoke (svg.clj:10)                                                                                                                                                      clj_pdf.section.svg$_render.invokeStatic (svg.clj:28)                                                                                                                                                 clj_pdf.section.svg$_render.invoke (svg.clj:24)                                                                                                                                                       clj_pdf.section.svg$fn__15479.invokeStatic (svg.clj:36)                                                                                                                                               clj_pdf.section.svg/fn (svg.clj:32)                                                                                                                                                                   clojure.lang.MultiFn.invoke (MultiFn.java:239)                                                                                                                                                        clojure.lang.AFn.applyToHelper (AFn.java:160)                                                                                                                                                         clojure.lang.AFn.applyTo (AFn.java:144)                                                                                                                                                               clojure.core$apply.invokeStatic (core.clj:669)                                                                                                                                                        clojure.core$apply.invoke (core.clj:660)                                                                                                                                                              clj_pdf.section$make_section.invokeStatic (section.clj:43)                                                                                                                                            clj_pdf.section$make_section.invoke (section.clj:17)                                                                                                                                                  clj_pdf.core$append_to_doc.invokeStatic (core.clj:112)                                                                                                                                                clj_pdf.core$append_to_doc.invoke (core.clj:107)                                                                                                                                                      clj_pdf.core$add_item.invokeStatic (core.clj:484)
    clj_pdf.core$add_item.invoke (core.clj:480)
    clj_pdf.core$write_doc.invokeStatic (core.clj:514)
    clj_pdf.core$write_doc.invoke (core.clj:502)
    clj_pdf.core$pdf.invokeStatic (core.clj:571)
    clj_pdf.core$pdf.invoke (core.clj:559)
```

**Comment by jwr** [comment]
Following up with a solution — this happens when `openjdk-8-jre-headless` is used and one possible solution is to remove the line:

```
assistive_technologies=org.GNOME.Accessibility.AtkWrapper
```

from `/etc/java-8-openjdk/accessibility.properties`.

Leaving this here so that if someone searches, this can be found.

**Comment by yogthos** [comment]
Ah good to know, might start a troubleshooting section in the readme for these kinds of quirks. Thanks for following up with the solution.
